elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/core/JoinPath.java

```diff
@@ -100,7 +101,14 @@ private PathElement resolvePathAttribute(Type<?> entityClass,
         if (dictionary.isAttribute(entityClass, fieldName)
```

Comment:
Make Path.resolvePathAttribute protected and override it here.
```java
    @Override
    protected PathElement resolvePathAttribute(Type<?> entityClass,
            String fieldName,
            String alias,
            Set<Argument> arguments,
            EntityDictionary dictionary) {
```

Reply:
Make Path.resolvePathElements protected and remove from JoinPath.

Reply:
:+1:

Reply:
updated.